src/ontology/edits/oeo-physical.omn

```diff
@@ -791,13 +794,13 @@ Class: EnergyCarrier
         <http://purl.obolibrary.org/obo/BFO_0000016>
     
     
-Class: EnergyCarrierQuantity
+Class: EnergyCarrierValue
 
     Annotations: 
-        <http://purl.obolibrary.org/obo/IAO_0000115> "An energy carrier quantity is a quantity about the amount of energy from a certain energy carrier."@en
+        <http://purl.obolibrary.org/obo/IAO_0000115> "An energy carrier value is a QuantityValue about the amount of energy from a certain energy carrier."@en
```

Comment:
What is meant by energy carrier quantity/value? The amount of an energy carrier ([kg]) or the (specific?) amount of (usable) energy of an energy carrier?

Reply:
If we stick with the -Values, shouldn't the value go to Energy instead of EnergyCarrier. The Energy is measured in Joule, kWh, therm, etc. And if you quantize in kg then it is the portionOfMatter but not the EnergyCarrier.